deployments/cardano-explorer-env-production.nix

```diff
@@ -0,0 +1,12 @@
+with (import ./../lib.nix);
+
+let
+  nodeProdConf = import ./../modules/cardano-node-prod.nix;
+in {
+  sl-explorer = nodeProdConf 40 region;
```

Comment:
It's `an arbitrary big number` :) I'm open to suggestions here. In future we might have two nodes for redundancy. And there needs to be some space for new nodes, although we could skip `40` for other nodes and go up.

Note that explorer node doesn't hold any stake and just provides a web interface to the node state.